implementations/wrapscan/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+pub mod wrap;
+pub use wrap::*;
+
+const DEFAULT_PROVIDER_URL: &str = "http/wraps.wrapscan.io";
+const RESOLVE_PATH: &str = "/r/";
+
+impl ModuleTrait for Module {
+    fn try_resolve_uri(
+        args: ArgsTryResolveUri,
+        env: Option<Env>,
+    ) -> Result<Option<UriResolverMaybeUriOrManifest>, String> {
+        if args.authority != "wrapscan" {
```

Comment:
NIT: We could add this to a const: `if args.authority != AUTHORITY`
Or something similar.
Personally, I like having all magic strings at the top of a file